src/zkevm_specs/evm_circuit/execution/create.py

```diff
@@ -130,71 +81,181 @@ def create(instruction: Instruction):
         all_but_one_64th_gas,
     )
 
-    # copy init_code from memory to bytecode
-    copy_rwc_inc, _ = instruction.copy_lookup(
-        instruction.curr.call_id,  # src_id
-        CopyDataTypeTag.Memory,  # src_type
-        instruction.next.code_hash,  # dst_id
-        CopyDataTypeTag.Bytecode,  # dst_type
-        offset,  # src_addr
-        offset + size,  # src_addr_boundary
-        FQ(0),  # dst_addr
-        size,  # length
-        instruction.curr.rw_counter + instruction.rw_counter_offset,
+    ### Do stack depth, nonce and balance pre-check
+    # ErrDepth constraint
+    is_error_depth, _ = instruction.compare(FQ(CALL_CREATE_DEPTH), depth, N_BYTES_STACK)
+    # ErrNonceUintOverflow constraint
+    is_nonce_overflow, _ = instruction.compare(nonce, nonce_prev, N_BYTES_U64)
```

Comment:
We need to compare `nonce_prev` with <code>2<sup>64</sup>-1</code> here (see https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/2d2bfc6ccf179ade1a8d063f9586b93e5283a557/zkevm-circuits/src/evm_circuit/execution/create.rs#L180).

Reply:
fixed in [22b6eff](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/416/commits/22b6effbb5dc11a274a9382e089bdc7d439bcb9e)